pkgs/development/python-modules/dinghy/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, aiofiles
+, aiohttp
+, click-log
+, emoji
+, glom
+, jinja2
+, pyyaml
+}:
+buildPythonPackage rec {
+  pname = "dinghy";
+  version = "0.13.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "nedbat";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-uRiWcrs3xIb6zxNg0d6/+NCqnEgadHSTLpS53CoZ5so=";
+  };
+
+  propagatedBuildInputs = [
+    aiofiles
+    aiohttp
+    click-log
+    emoji
+    glom
+    jinja2
+    pyyaml
+  ];
+
+  checkInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "dinghy.cli" ];
+
+  meta = with lib; {
+    description = "Dinghy daily digest tool";
```

Putting the word "github" in here would make it clearer for the uninitiated.

---

I just copied the description from here: https://github.com/nedbat/dinghy/blob/1ab5e4269973af2514d511c60790eb6c384f49d7/setup.cfg#L6

Nevertheless, I agree that the description isn't very good:

```suggestion
    description = "A GitHub activity digest tool";
```

---

Done!